modular_skyrat/modules/modular_weapons/code/company_and_or_faction_based/xhihao_light_arms/submachinegun.dm

```diff
@@ -0,0 +1,62 @@
+// Evil .585 smg that blueshields spawn with that will throw your screen like hell but itll sure kill whoever threatens a head really good
+
+/obj/item/gun/ballistic/automatic/xhihao_smg
+	name = "\improper Xhihao 'Bogseo' Submachinegun"
+	desc = "A weapon that could hardly be called a 'sub' machinegun, firing the monstrous .585 cartridge. \
+		It provides enough kick to bruise a shoulder pretty bad if used without protection."
+
+	icon = 'modular_skyrat/modules/modular_weapons/icons/obj/company_and_or_faction_based/xhihao_light_arms/guns32x.dmi'
+	icon_state = "bogseo"
+
+	lefthand_file = 'modular_skyrat/modules/modular_weapons/icons/mob/company_and_or_faction_based/xhihao_light_arms/guns_lefthand.dmi'
+	righthand_file = 'modular_skyrat/modules/modular_weapons/icons/mob/company_and_or_faction_based/xhihao_light_arms/guns_righthand.dmi'
+	inhand_icon_state = "bogseo"
+
+	special_mags = FALSE
+
+	bolt_type = BOLT_TYPE_STANDARD
+
+	w_class = WEIGHT_CLASS_BULKY
+	weapon_weight = WEAPON_HEAVY
+	slot_flags = ITEM_SLOT_OCLOTHING | ITEM_SLOT_BELT
+
+	accepted_magazine_type = /obj/item/ammo_box/magazine/c585trappiste_pistol
+
+	fire_sound = 'modular_skyrat/modules/modular_weapons/sounds/smg_heavy.ogg'
+	can_suppress = TRUE
+
+	can_bayonet = FALSE
+
+	suppressor_x_offset = 9
+
+	burst_size = 1
+	fire_delay = 0.15 SECONDS
+
+	// Makes the bullets do 40 damage
+	projectile_damage_multiplier = 0.9
```

Comment:
I'd rather we don't have projectile damage multipliers on the weapons if it can be avoided.
I don't want players to have to think "oh this gun is going to do less damage per bullet, for the same exact kind of bullet", I'd rather we have bullets be the determinant of how much damage something deals.

Reply:
second this
That and while what I have to say won't really matters- I think it's worth noting that people looking at the gun in the game won't really know this thing does less damage, and, if so, by how much.

Knowing how much damage a gun will do is something that I think is fairly important